drivers/Makefile.include

```diff
@@ -408,10 +408,18 @@ ifneq (,$(filter w5100,$(USEMODULE)))
   USEMODULE_INCLUDES += $(RIOTBASE)/drivers/w5100/include
 endif
 
+ifneq (,$(filter wiegand,$(USEMODULE)))
+  USEMODULE_INCLUDES += $(RIOTBASE)/drivers/wiegand/include
+endif
+
 ifneq (,$(filter ws281x,$(USEMODULE)))
   USEMODULE_INCLUDES += $(RIOTBASE)/drivers/ws281x/include
 endif
 
 ifneq (,$(filter xbee,$(USEMODULE)))
   USEMODULE_INCLUDES += $(RIOTBASE)/drivers/xbee/include
 endif
+
+ifneq (,$(filter xbee,$(USEMODULE)))
+  USEMODULE_INCLUDES += $(RIOTBASE)/drivers/xbee/include
+endif
```

Comment:
This is unrelated.

Reply:
Removed, I think it was automatically merged at some point. In any-case latest master branch removes this file.